samples/matter/common/src/bridge/bridge_manager.cpp

```diff
@@ -31,10 +31,12 @@ void BridgeManager::Init()
 				     false);
 }
 
-CHIP_ERROR BridgeManager::AddBridgedDevices(BridgedDevice *aDevice, BridgedDeviceDataProvider *aDataProvider)
+CHIP_ERROR BridgeManager::AddBridgedDevices(BridgedDevice *aDevices[], BridgedDeviceDataProvider *aDataProvider,
```

Comment:
```suggestion
CHIP_ERROR BridgeManager::AddBridgedDevices(BridgedDevice *Devices[], BridgedDeviceDataProvider *DataProvider,
```

NIT: I haven't spotted it recently, but I guess we should stick to the same convention in all methods and AFAIR we decided to not use `a` prefixes for methods' arguments.